Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.5
+
+# USER ubuntu
+
+WORKDIR /usr/src/app
+
+# Copy source files from the host to the container's workdir
+COPY . .
+
+# Install NLTK package
+RUN pip install .
+RUN pip install tox
+
+RUN source tools/travis/pre-install.sh
+RUN chmod +x tools/travis/coverage-pylint.sh
+RUN chmod +x tools/travis/third-party.sh
+
+# Keep the container running
+CMD tail -f /dev/null
```

Comment:
If there is a need for this CMD in the first place, [JSON-style](https://github.com/hadolint/hadolint/wiki/DL3025) is usually preferable:

```
CMD ["tail", "-f", "/dev/null"]
```